helmfile/charts/opensearch/configurer/files/index-templates/kubernetes.template.json

```
@@ -7,6 +7,16 @@
       "index.refresh_interval": "10s",
       "plugins.index_state_management.rollover_alias": "kubernetes",
       "mapping.total_fields.limit": "1000"
+    },
+    "mappings" : {
+      "properties" : {
+        "ts" : {
+          "type" : "text"
+        },
+        "@timestamp" : {
+          "type" : "date"
```

NIT, swap places please :)
```suggestion
      "properties" : {
        "ts" : {
          "type" : "text"
        },
        "@timestamp" : {
          "type" : "date"
```